organizations/payments.go

```diff
@@ -325,3 +331,68 @@ func (oh *OrganizationHandler) CreateCheckoutSession(w http.ResponseWriter, r *h
 
 	utils.GetSuccess("successfully initiated payment", s.URL, w)
 }
+
+// AddCard encapsulates operations for creating, deleting and querying cards using the Stripe REST API.
+func (oh *OrganizationHandler) AddCard(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+
+	MemberID := mux.Vars(r)["mem_id"]
+	objID, err := primitive.ObjectIDFromHex(MemberID)
+
+	if err != nil {
+		utils.GetError(errors.New("invalid id"), http.StatusBadRequest, w)
+		return
+	}
+
+	member, _ := utils.GetMongoDbDoc(MemberCollectionName, bson.M{"_id": objID})
```

Comment:
Why are you not using the organization id in this function?
r.HandleFunc("/organizations/members/{mem_id}/addcard
This doesn't seem right. And not in line with our endpoints design. Please adjust
It should be r.HandleFunc("/organizations/{id}/members/{mem_id}/addcard